src/app/home/page.tsx

```diff
@@ -1,14 +1,32 @@
 "use client";
+import { BasicRoundedButton } from "@/components/buttons/basic-rounded-button/Basic-rounded-button";
 import { useAuthSession } from "@/lib/contexts/auth-context/auth-context";
+import { Link } from "@mui/material";
 import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import moduleStyles from "../home-page.module.css";
 
 export default function LandingPage() {
+  // State to hold the current date
+  const [currentDate, setCurrentDate] = useState("");
+
+  // useEffect to set the current date when the component mounts
+  useEffect(() => {
```

Comment:
Instead of using a `useEffect`, you could simply set the current date as the initial value in the useState statement

Reply:
I used state since the date changes at midnight, but I guess it makes more sense to prevent the date from changing at the time of creating the log. I'll update this!

Reply:
> I used state changes since the date updates at midnight, but let me know if you disagree or if I'm using it wrong 🤔. I just assumed this date was just for display, but I think eventually the user should be able to update the date of their log formData in case the date changes to the next day.
> 
> Also, I see what you're saying about creating a function for calculating the date to prevent repeating code. I'll update this!

Yeah I think we don't need state for this

Reply:
I removed the state and made a function getCurrentDate() that gets called in the HTML